src/components/Stack/README.md

```diff
@@ -138,6 +138,19 @@ Use for aligning buttons or secondary content to the right edge of another eleme
 
 ---
 
+## Stack item
+
+Use item to align buttons or secondary content to the right edge of another element, allowing it to wrap below on small screens.
```

Comment:
The `Stack` automatically wraps each of its children in a `Stack.Item` (unless the implementor has already wrapped a child/children with `Stack.Item`). It may be helpful to mention that, and then explain that you can use the subcomponent directly with the `fill` prop when you want the item to fill the available horizontal space in the stack (and mention they can see this in action in the "Stack where a single item fills the remaining space" example above).